src/smc-project/browser-websocket/api.ts

```diff
@@ -67,8 +72,13 @@ async function listing(path: string, hidden?: boolean): Promise<object[]> {
 }
 
 import { run_prettier } from "../prettier";
-async function prettier(client: any, path: string, options: any): Promise<any> {
-  return await run_prettier(client, path, options);
+async function prettier(
+  client: any,
+  path: string,
+  options: any,
+  logger: any
+): Promise<any> {
+  return await run_prettier(client, path, options, logger);
```

Comment:
This is a function that does NOTHING.  WEIRD.

Reply:
yes, it was there like that before I added this. However, when I merged the one for jupyter cells, I removed it. see [merge commit](https://github.com/sagemathinc/cocalc/pull/3113/files#diff-05af434d8b71074e5c913dc6fad6d8dcR45)